java/arcs/core/storage/database/DatabaseManager.kt

```diff
@@ -56,6 +56,15 @@ interface DatabaseManager {
 
     /** Garbage collection run: removes unused entities. */
     suspend fun runGarbageCollection()
+
+    /** Gets the count of entities stored. */
+    suspend fun getEntitiesCount(inMemory: Boolean): Long
+
+    /** Gets the size of the total storage used in bytes. */
+    suspend fun getStorageSize(inMemory: Boolean): Long
+
+    /** Returns if the current storage is too large. */
```

Comment:
can you provide more detail on what it means to for storage to be too large.
for example 
/** Returns if the current storage is too large (defined as any of the databases being larger than a threshold). */
maybe also move the threshold in this file.

Reply:
I changed the description. I'll leave the threshold where it is so we don't change too many things in one PR.